spec/factories/pathfinder_class.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :pathfinder_class do
+    sequence(:name) { |n| "Pathfinder Class name 0#{n}" }
```

Comment:
This will need to be updated with the validations

Reply:
Corrected with `.sample`